### PR TITLE
DOC-2411: Add TINY-10942 release note entry

### PR DESCRIPTION
**Pull request by FarzadHayat**
Ticket: DOC-2411

Site: [Staging branch](http://docs-feature-711-doc-2411tiny-10942.staging.tiny.cloud/docs/tinymce/latest/7.1.1-release-notes/#autocompleter-possible-values-are-no-longer-capped-at-a-length-of-10)

Changes:
* DOC-2411: Add TINY-10942 release note entry

Pre-checks:
- [x] Branch prefixed with `feature/<version>/`, `hotfix/<version>/`, `staging/<version>/`, or `release/<version>/`
- [-] `modules/ROOT/nav.adoc` has been updated `(if applicable)`
- [-] Files has been included where required `(if applicable)`
- [-] Files removed have been deleted, not just excluded from the build `(if applicable)`
- [x] Files added for `New product features`, and included a `release note` entry.

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/7.1.1-release-notes.adoc [hunk 1]
```diff
@@ -155,6 +155,13 @@ For information on using Enhanced Skins & Icon Packs, see: xref:enhanced-skins-a
 
 {productname} {release-version} also includes the following bug fix<es>:
 
+=== Autocompleter possible values are no longer capped at a length of 10.
+// #TINY-10942
+
+Previously in {productname}, the Autocompleter option values were constrained to lengths of less than 10. As a consequence, the Autocompleter popover would close after only one value being selected if the selected value was greater than or equal to 10. This created a limitation for users who needed to select multiple values consecutively with lengths greater than 10.
```
Comment (FarzadHayat) [inline modules/ROOT/pages/7.1.1-release-notes.adoc:161]: @danoaky-tiny do we want to include any detail here about why there was previously a hardcoded limit of 10 characters on the option values?

Comment (danoaky-tiny) [reply modules/ROOT/pages/7.1.1-release-notes.adoc]: You could mention that it was a regression. Originally the limit of 10 was a used as a cap to only _match_ the first 10 characters, but still show the menu. Starting in v6.0.0, it no longer showed the menu after 10 characters.

**Review by danoaky-tiny — APPROVED** [review]

**Review by kemister85 — APPROVED** [review]

File: modules/ROOT/pages/7.1.1-release-notes.adoc (hunk 1 above)
Comment (FarzadHayat) [reply modules/ROOT/pages/7.1.1-release-notes.adoc]: That would just redirect the question to why it stopped working from v6.0.0. I think it's too much to explain in a release note at that point so I will just keep the entry as is.

**Review by sabisabisabisab — COMMENTED** [review]
🥂

**Pull request merged**